Microsoft.Toolkit/Extensions/TypeExtensions.cs

```diff
@@ -113,5 +121,28 @@ static string FormatDisplayString(Type type)
             // be removed once this issue is resolved: https://github.com/dotnet/roslyn/issues/5835.
             return DisplayNames.GetValue(type, t => FormatDisplayString(t));
         }
+
+#if NETSTANDARD1_4
+        /// <summary>
+        /// Returns an array of types representing the generic arguments.
+        /// </summary>
+        /// <param name="type">The input type.</param>
+        /// <returns>An array of types representing the generic arguments.</returns>
+        public static Type[] GetGenericArguments(this Type type)
+        {
+            return type.GetTypeInfo().GenericTypeParameters;
+        }
+
+        /// <summary>
+        /// Returns whether <paramref name="type"/> is an instance of <paramref name="value"/>.
+        /// </summary>
+        /// <param name="type">The input type.</param>
+        /// <param name="value">The type to check against.</param>
+        /// <returns>True if it is <paramref name="type"/> is an instance of <paramref name="value"/> false if not.</returns>
```

Comment:
Couple minor nits:
-  Use the `langword` XML tag for keywords in comments
- There was one small grammar mistake

```suggestion
        /// <returns>Returns <see langword="true"/> if <paramref name="type"/> is an instance of <paramref name="value"/>, <see langword="false"/> otherwise.</returns>
```